src/front/spv/mod.rs

```diff
@@ -817,18 +817,16 @@ impl<I: Iterator<Item = u32>> Parser<I> {
                     match type_arena[image_type_handle].inner {
                         //TODO: compare the result type
                         crate::TypeInner::Image {
-                            base: _,
+                            kind: _,
```

Comment:
Nit: can't this be changed to using the syntax with dots? e.g. :

```rust
crate::TypeInner::Image {
    dim,
    arrayed,
    class: crate::ImageClass::Sampled,
    ..
} => {
```

Reply:
I believe explicit syntax `kind: _` is superior. If new fields are added to the struct, the author would have to revisit these places and decided if these fields need to be considered.